src/scenarios/mauiblazorios/pre.py

```diff
@@ -72,8 +72,8 @@
 remove_aab_files(output_dir)
 
 # Copy the MauiVersion to a file so we have it on the machine
-maui_version = get_version_from_dll_powershell_ios(rf"./{const.APPDIR}/obj/Release/{precommands.framework}/ios-arm64/ipa/Payload/{EXENAME}.app/Microsoft.Maui.dll")
+maui_version = get_version_from_dll_powershell_ios(rf"./{const.APPDIR}/obj/Release/net8.0-ios/ios-arm64/linked/Microsoft.Maui.dll")
```

Comment:
nit: I know this already got merged in, so for future reference it might be better to use `{precommands.framework}` instead of explicit `net8.0-ios` tfm

Reply:
Yes we missed that, sorry. Follow-up PR in https://github.com/dotnet/performance/pull/3154.